File: 2998-count-symmetric-integers/count-symmetric-integers.go
func countSymmetricIntegers(low int, high int) int {
    count := 0
    for i:= low; i < high+1;i++ {
        check := fmt.Sprintf("%d",i)
        if len(check) % 2 == 1{
            continue 
        } 
            mid := len(check) / 2
            sum1 := 0
            sum2 := 0 
            for j:= 0; j < mid; j++ {
                sum1+= int(check[j] - '0')
            }
            for j:= mid; j < len(check); j++ {
                sum2+= int(check[j] - '0')
            }
            if sum1 == sum2 {
                count++
        }
    }    
    return count
}  

File: 2998-count-symmetric-integers/count-symmetric-integers.py
class Solution:
    def countSymmetricIntegers(self, low: int, high: int) -> int:
        count = 0
        for i in range(low,high+1):
            string = str(i)
            if len(string) % 2 == 1:
                continue
            mid = len(string) //2
            left = string[:mid]
            right = string[mid:]
            left_sum = 0
            right_sum = 0
            for num in left:
                left_sum += int(num)
            for num in right:
                right_sum += int(num)
            if left_sum == right_sum:
                count+=1
        return count


        